Makefile

```diff
@@ -21,7 +21,10 @@ RUBY_ENV += RUBY_GC_OLDMALLOC_LIMIT=40125988
 RUBY_ENV += RUBY_GC_OLDMALLOC_LIMIT_MAX=72226778
 RUBY_ENV += RUBY_GC_OLDMALLOC_LIMIT_GROWTH_FACTOR=1.2
 
-
+container_runtime ?= $(shell docker --version | grep -q podman && echo podman || echo docker)
+oracle_podman_flags ?= --mount=type=bind,source=$(ORACLE_DATA_DIR)/oracle-database,destination=/mnt,relabel=shared
+oracle_docker_flags ?= --security-opt apparmor=docker-default \
```

Comment:
I was working on it, the apparmor thing should be tested as well.

Reply:
The apparmor opt failed for me. Apparmor not installed on Fedora by default and I don't see why need it on Fedora where containers are isolated by selinux. But I assumed that it is working with docker somehow. Is there any command to check for apparmor presence so that this option is only passed when it indeed is installed? I can add such a check.

Reply:
I mean it should be tested (conditional `if`) ... this was something workingon Fedora 32 and other distributions (like Ubuntu)

Reply:
I would like to test it. But how exactly to test presence of it? On Fedora 34 I see:
```
Error: Apparmor profile "docker-default" specified, but Apparmor is not enabled on this system
make: *** [Makefile:112: oracle-database] Error 125
```

Do you know a command to check whether apparmor is enabled?